peach_tuntiwong/week01/day2/js/main.js

```diff
@@ -0,0 +1,101 @@
+var solution = ['F', 'O', 'X'];
+var currentGuess = ['_', '_', '_'];
+
+var checkLetter = function(letter) {
+  var inputLetter = letter.toUpperCase(); // Set all input to uppercase
+  if (inputLetter == 'F') { // Return 1 if guess correct, return 0 if guess wrong
+    currentGuess[0] = inputLetter;
+    return 1;
+  } else if (inputLetter == 'O') {
+    currentGuess[1] = inputLetter;
+    return 1;
+  } else if (inputLetter == 'X') {
+    currentGuess[2] = inputLetter;
+    return 1;
+  } else {
+    return 0;
+  }
```

Comment:
We ideally want to use a `for` loop here to iterate over each letter of the mystery word and compare it to the guess letter; this will work for any word, whereas yours will only work for FOX (and it will get longer if you change it to test for a longer word).